cmd/goss/goss.go

```diff
@@ -230,14 +230,14 @@ func main() {
 					Name:  "package",
 					Usage: "add new package",
 					Action: func(c *cli.Context) error {
-						return goss.AddResources(c.GlobalString("gossfile"), "Package", c.Args(), newRuntimeConfigFromCLI(c))
+						return goss.AddResources(c.GlobalString("gossfile"), "Packages", c.Args(), newRuntimeConfigFromCLI(c))
```

Comment:
`"Packages"` 

This is where I get uncomfortable with magic strings. If you are refactoring this code section it should be converted to an enum like of resources. Which would work well with the map of resource "appenders"